### PR TITLE
Create ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60252](https://cumulocity.atlassian.net/browse/MTM-60252)
Original PR: [6927](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6927)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added a fix for official plugins ribbon not being shown. (#6839) [GRAFT][release/cd] (#6927)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix missing official plugins ribbon in marketplace
```

File: content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60252
+version: 1020.18.4
+---
+Added a fix for official plugins ribbon not being shown. (#6839) [GRAFT][release/cd] (#6927)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md:17]: The Claude AI suggested to use the following body:
```suggestion
The Camunda Platform marketplace displays a ribbon to highlight official plugins developed by Camunda. Due to a bug, this ribbon was not shown for certain plugins, even though they were official Camunda plugins. This issue has now been fixed. Users will now correctly see the "Official Plugin" ribbon for all plugins developed by Camunda, making it easier to identify them in the marketplace.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md (hunk 2 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md:17]:
```suggestion

```In the extensions page a ribbon is used to highlight official plugins. Due to a bug, this ribbon was not shown for certain plugins, even though they were official. This issue has now been fixed.

File: content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md (hunk 1 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md:3]:
```suggestion
title: Official plugins now have the correct ribbon displayed.
```

File: content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+date: ""
+title: Official plugins now have the correct ribbon displayed.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md:3]:
```suggestion
title: Ribbon correctly highlights all official plugins in Extensions page
```

**Comment by BeateRixen** [comment]
Changed the label to Administration as this is actually happening in the Administration application

File: content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60252
+version: 1020.18.4
+---
+
+In the **Extensions** page in the **Ecosystem** a ribbon is used to highlight official plugins. This ribbon was not shown for certain plugins, even though they were official. This issue has now been fixed. The ribbon is displayed for all official plugins in the **Extensions** page.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md:18]:
```suggestion
On the **Extensions** page in the **Ecosystem** menu a ribbon is used to highlight official plugins. This ribbon was not shown for certain plugins, even though they were official. This issue has now been fixed. The ribbon is displayed for all official plugins on the **Extensions** page.
```

File: content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+date: ""
+title: Ribbon correctly highlights all official plugins in Extensions page
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-18-4-added-a-fix-for-official-plugins-ribbon-not-being-shown-6839.md:3]:
```suggestion
title: Plugins now shown with correct type "official" on the Extensions page
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**